src/Tokenizer.js

```diff
@@ -457,52 +457,52 @@ module.exports = class Tokenizer {
   link(src) {
     const cap = this.rules.inline.link.exec(src);
     if (cap) {
-      const trimmedUrl = cap[2].trim();
-      if (!this.options.pedantic && trimmedUrl.startsWith('<')) {
-        // commonmark requires matching angle brackets
-        if (!trimmedUrl.endsWith('>')) {
-          return;
-        }
-
-        // ending angle bracket cannot be escaped
-        const rtrimSlash = rtrim(trimmedUrl.slice(0, -1), '\\');
-        if ((trimmedUrl.length - rtrimSlash.length) % 2 === 0) {
-          return;
-        }
-      } else {
+      let href = cap[2].trim();
+      if (!href.startsWith('<')) {
         // find closing parenthesis
```

Comment:
this block needs to run if it is pedantic as well

Reply:
The "findClosingBracket" block does run for both pedantic and not. This occurs for all incoming code, and then the pedantic-specific items run after in their own block.

The original logic runs this if `!pedantic && startsWith('<')` fails. This is logically identical to: if `pedantic || !startswith('<')`, so that's essentially what this is doing now.